### PR TITLE
Add docs for local kubeflow + local metadata store

### PR DESCRIPTION
**Pull request by schustmi**
## Describe changes

Based on a Slack-Thread, this PR adds information on how to access the local metadata store with local kubeflow.

## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [x] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/advanced-guide/integrations) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Other (add details above)

### CODE REVIEW

File: docs/book/mlops-stacks/orchestrators/kubeflow.md
Comment (stefannica) [inline docs/book/mlops-stacks/orchestrators/kubeflow.md:140]: Doing this on the local filesystem basically gives everyone on the machine read/write access to all these files in your home directory (on Linux at least), which includes your local secrets manager stores.

**Review by stefannica — CHANGES_REQUESTED** [review]
Didn't we already cover this use-case by using the local UID/GID in the container image built locally for Kubeflow ?

File: docs/book/mlops-stacks/orchestrators/kubeflow.md
Comment (fa9r) [inline docs/book/mlops-stacks/orchestrators/kubeflow.md:136]:
```suggestion
database is writable from inside the Kubeflow Pods. To do so, first figure out where
```

**Review by fa9r — CHANGES_REQUESTED** [review]
As @stefannica pointed out, having to change the global config access is somewhat unsafe und definitely feels hacky. Since we're reworking the metadata store anyways, I would propose to hold off on these docs changes, and, if needed, document how to do use local Kubeflow under the new structure later.

File: docs/book/mlops-stacks/orchestrators/kubeflow.md
Comment (schustmi) [reply docs/book/mlops-stacks/orchestrators/kubeflow.md:140]: The user in slack still ran into issues, even with the UID/GID commands in the image. It might not be related to Kubeflow after all, as there seems to be some SQLite issues when the user doesn't have write access to the entire directory containing the database.
I guess this call could also be limited to exactly this directory by doing `chmod -R a+rw <GLOBAL_CONFIG_DIR>/local_stores/<UUID>`, which only contains the sqlite database. However I also don't have a problem not adding that at all if you think it's not a common error

**Pull request closed without merging**
